src/Sylius/Bundle/ShopBundle/Resources/views/Currency/selector.html.twig

```diff
@@ -0,0 +1,13 @@
+<div class="ui pointing dropdown link item" id="currency-selector">
```

Comment:
CSS namespaces? Sounds fun! 

![comic-adult-swim-this-is-fine-3o6UBpHgaXFDNAuttm](https://camo.githubusercontent.com/4aa60f5e848a51db3ea7d677c451daf71564418b/68747470733a2f2f6d65646961322e67697068792e636f6d2f6d656469612f336f365542704867615846444e417574746d2f67697068792e676966>)